light-table.yml

```diff
@@ -0,0 +1,11 @@
+name: Light Table
+desc: Light Table is a next generation code editor that connects you to your creation with instant feedback. Light Table is very customizable and can display anything a Chromium browser can.
+site: http://lighttable.com/
+tags:
+- ide
+- clojure
+- clojurescript
+- electron
+upforgrabs:
+    name: good first issue
```

Comment:
It should be prepended by 2 spaces not 4. I don't know if it will make a difference though.

Reply:
Okay, I'll fix that in my PR to up-for-grabs.